guides/development-workflow.md

```diff
@@ -19,46 +19,52 @@ title: Development workflow
 
 ### Development principles
 
-As part of regular retrospectives we evolve our principles by identifying what
-works well. When we try something that helps with a particular pattern of
-problem the team can acknowledge the benefits it brings and introduce it as a
-principle.
-
 #### Communicate frequently
 
-- space to learn from each other
+- learn from each other
 - make shared decisions
 - prevent duplication
 - solve problems quicker
 
 #### Two pairs of eyes
 
-All code we write needs be reviewed by another developer. We hope that doing
-this will mean we:
+All code we write should be reviewed by at least one peer
 
-- ship better quality code
-- get the opportunity to learn new skills
-- increase the shared knowledge of our products
+- build shared understanding
+- build shared knowledge of code bases
+- ship safer and higher quality code
+- learn new skills and techniques
 
-#### Everything is tracked
+#### Share our progress
 
-All code we write is represented in Trello (or another ticketing system as
-required by the client) so that:
+All code we write is represented in Trello (or similar) so that we can:
```

Comment:
I think it's good practice for it to be true for projects where multiple people are working together on something. Without it, the product management and prioritisation is basically impossible.

I think Trello only being used for active client projects is fine. Support work is represented by Zendesk tickets, which I think is fine. Internal projects often have Trello boards. I'd like to explore using the task management within Productive for those at some point.

Maybe it's "most code" instead of "all code"?

Reply:
👍 I have updated this to be more explicit that it's important for "teams sprinting on client work" to use Trello however support work uses Zendesk.